src/systems/a320_systems/src/hydraulic/mod.rs

```diff
@@ -2381,9 +2383,10 @@ impl A320RamAirTurbineController {
         let solenoid_2_should_trigger_deployment_if_powered =
             emergency_elec_state.is_in_emergency_elec() || rat_and_emer_gen_man_on.is_pressed();
 
-        self.should_deploy = (self.is_solenoid_1_powered
-            && solenoid_1_should_trigger_deployment_if_powered)
-            || (self.is_solenoid_2_powered && solenoid_2_should_trigger_deployment_if_powered);
+        // due to initialization issues the RAT will not deployed in any case when simulation has just started
+        self.should_deploy = (context.simulation_time() >= 2.)
```

Comment:
Might be a nitpick but couldn't we make that "context.simulation_time() >= 2"  a method like context.is_sim_initialiazing() ?
So any function that just need a simple disable on iinit can use this, and if something changes in the sim and the 2 becomes a 4 seconds, we can just change this at one place.

Reply:
For what it's worth, in JS instruments one can query the GameState which maybe can be used to set the LVAR (e.g. https://github.com/flybywiresim/a32nx/blob/56c798eb25c60ab66500f2a6023333adff6c70d4/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_MainDisplay.js#L264)

Reply:
Thank you for the tips, I think it's much better now!